pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -16,13 +16,13 @@ limitations under the License.
 
 // This file contains structures that implement scheduling queue types.
 // Scheduling queues hold pods waiting to be scheduled. This file implements a
-// priority queue which has two sub queues. One sub-queue holds pods that are
-// being considered for scheduling. This is called activeQ. Another queue holds
-// pods that are moved from unschedulablePods and will be moved to active queue
-// when backoff are are completed. The latter is called backoffQ. In addition
-// to the two sub-queues, there is a data structure in priority queue called
-// unschedulablePods holds pods that are already tried and are determined to
-// be unschedulable.
+// priority queue which has two sub queues and a additional data structure.
```

Comment:
This comment was not very well structured in the past :sweat_smile: 
It's better to state the names at the beginning and then expand. Something like this:

This file has a priority queue which has two sub-queues and an additional data structure, namely: activeQ, backoffQ and unschedulablePods.
- activeQ holds pods that are being considered for scheduling.
- backoffQ holds pods that moved from unschedulablePods and will move to activeQ when their backoff periods complete
- unschedulablePods holds pods that were already attempted for scheduling and are currently determined to be unschedulable.

Reply:
Yeah. The original comment is very long and not well structured.
Updated.